guides/common/modules/proc_customizing-the-discovery-pxe-boot.adoc

```diff
@@ -0,0 +1,45 @@
+[id="Customizing_the_Discovery_PXE_Boot_{context}"]
+= Customizing the Discovery PXE boot
+[[Discovery_Templates_and_Snippets_Settings_{context}]]
+[[Provisioning_Template_PXELinux_Discovery_Snippet_{context}]]
+
+{Project} builds PXE boot menus from the global provisioning templates: `PXELinux global default` for BIOS provisioning, `PXEGrub global default`, and `PXEGrub2 global default` for UEFI provisioning.
+The PXE boot menus are available on {ProjectServer} and all {SmartProxies} that have the TFTP feature enabled.
+
+The Discovery menu item uses a Linux kernel for the operating system, which passes kernel parameters to configure the Discovery service.
+You can customize the passed kernel parameters by changing the following snippets:
+
+Snippet `pxelinux_discovery`:: This snippet is included in the `PXELinux global default` template.
++
+This snippet renders the Discovery boot menu option.
+The `KERNEL` and `APPEND` options boot the Discovery kernel and initial RAM disk.
+The `APPEND` option contains kernel parameters.
+
+Snippet `pxegrub_discovery`:: This snippet is included in the `PXEGrub global default` template.
+However, Discovery is http://projects.theforeman.org/issues/15997[not supported with GRUB 1.x].
```

Comment:
Discovery doesn't work with Grub 1.x. This sounds like it's suggesting the other way around.

Reply:
In that case, I would word it differently because "not supported with" sounds like a grammatical error.

Suggeston: "Discovery is incompatible with GRUB 1.x".

Reply:
How about "Discovery is not implemented for GRUB 1.x?"